### PR TITLE
Is this commit still relevant

### PR DESCRIPTION
**Pull request by frcroth**
Looks unmerged

### CODE REVIEW

**Comment by sirkrypt0** [comment]
Nice, I think we are definitely missing those

File: packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testUS60disableAutocompletion.st [hunk 1]
```diff
@@ -0,0 +1,7 @@
+tests
+testUS60disableAutocompletion
+
+	Preferences setFlag: self autocompletionPreference
+						toValue: false
+						during: [
+							self deny: self editorModel wantsAutocompletion]
```
Comment (sirkrypt0) [inline packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testUS60disableAutocompletion.st]: The indentation seems far to much

File: packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testUS60enableAutocompletion.st [hunk 2]
```diff
@@ -0,0 +1,7 @@
+tests
+testUS60enableAutocompletion
+
+	Preferences setFlag: self autocompletionPreference
+						toValue: true
+						during: [
+							self assert: self editorModel wantsAutocompletion]
```
Comment (sirkrypt0) [inline packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testUS60enableAutocompletion.st]: Indentation

**Review by sirkrypt0 — CHANGES_REQUESTED** [review]

**Comment by konradh** [comment]
Yes, it is a acceptance test for the disabling autocompletion functionality and still relevant.

**Review by sirkrypt0 — APPROVED** [review]

**Pull request merged**